extension/types/hiraganaLookupMap.ts

```diff
@@ -0,0 +1,247 @@
+export const kanaToHiraganaNormalizationMap: Record<string, string> = {
+  ぁ: 'ぁ',
```

Comment:
As I mentioned elsewhere, what about just removing the 1:1 mappings here since we already handle that at the call site.

Right now it's a lot to sift through to see which things are actually being changed.